src/datatablerow/datatablerow.jl

```diff
@@ -4,10 +4,6 @@ immutable DataTableRow{T <: AbstractDataTable}
     row::Int
 end
 
-function Base.getindex(r::DataTableRow, idx::AbstractArray)
```

Comment:
I think this function is still needed, though ideally it would be tested: it's different from the next one, which applies when passing a single column index. Since it's not added by this PR, it's OK to leave it untested for now (unless tests where removed by the PR).